bin/get_input_data.R

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env Rscript 
+
+# extract reference and query data to be used in the control workflow
+# required data include:
+# - reference expression matrix (10X dir)
+# - query expression matrix 
+# - reference SDRF file (need cell ids, inferred cell types, and CL terms)
+# - marker genes file for reference dataset 
+
+suppressPackageStartupMessages(require(optparse))
+suppressPackageStartupMessages(require(workflowscriptscommon))
+suppressPackageStartupMessages(require(R.utils))
+
+option_list = list(
+    make_option(
+        c("-a", "--reference-mat-url"),
+        action = "store",
+        default = NA,
+        type = 'character',
```

Comment:
Why no help text?

Reply:
forgot to add these fields, will fix now